src/overcooked_ai_py/visualization/visualization_utils.py

```diff
@@ -0,0 +1,55 @@
+import tempfile, webbrowser, json, uuid, os
+from IPython.display import display, HTML
+from string import Template
+from overcooked_ai_py.static import VISUALIZATION_DIR
+
+def load_visualization_file_as_str(filename):
+    with open(os.path.join(VISUALIZATION_DIR, filename), 'r') as f:
+        lines = f.readlines()
+        return "\n".join(lines)
+
```

Comment:
I don't like how this code (and its previous version based on pkgutil) is failing on osx test workflow despite other reading files work without problems. I will try to fix it later but if you have any idea why reading file can fail, but only on osx workflow let me know.

```
Traceback (most recent call last):
  File "testing/visualization_test.py", line 27, in test_create_chart_html
    html = create_chart_html(self.extracted_events1, box_id=box_id)
  File "/Users/runner/hostedtoolcache/Python/3.7.8/x64/lib/python3.7/site-packages/overcooked_ai_py/visualization/visualization_utils.py", line 48, in create_chart_html
    js_text_template = Template(load_visualization_file_as_str("event_chart.js"))
  File "/Users/runner/hostedtoolcache/Python/3.7.8/x64/lib/python3.7/site-packages/overcooked_ai_py/visualization/visualization_utils.py", line 7, in load_visualization_file_as_str
    with open(os.path.join(VISUALIZATION_DIR, filename), 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/runner/hostedtoolcache/Python/3.7.8/x64/lib/python3.7/site-packages/overcooked_ai_py/visualization/event_chart.js'
```

Reply:
solved, it was about -e flag in pip install